packages/storybook/src/stories/ino-select/ino-select.stories.ts

```diff
@@ -2,9 +2,16 @@ import { Components } from '@inovex.de/elements';
 import { useEffect } from '@storybook/client-api';
 import { Story } from '@storybook/web-components';
 import { html } from 'lit-html';
-import { defaultDecorator } from '../utils';
+import { defaultDecorator, showSnackbar } from '../utils';
 import './ino-select.scss';
 
+const handleFormSubmission = (e) => {
```

Comment:
Form submit not workings as expeckted. When I click on the button, an empty snackbar opens (In both cases - empty submit & filled submit)

Reply:
Huh, I can't really reproduce the issues you are seeing. Did you start the storybook with `yarn dev` inside the storybook package by any chance? In that case, make sure that you've run `yarn build` inside the elements package before. You can also just use the `yarn start` script on root level which combines these two steps.

Reply:
I retried it with your instructions and now it looks good and everything works fine. I think something was not build correctly.